cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -37,14 +45,19 @@ def delete_all_vm_tasks():
     sel.handle_alert()
 
 
-def check_log(log, verify_tags):
-    """Verify that verify_tags are shown in the log as the pattern below"""
-    for tag in verify_tags:
-        assert re.findall(r'\n(.+Scanning::Job#{}.+)\n'.format(re.escape(tag)), log)
+def verify_log(log, verification_tags):
+    """Verify that verification_tags are shown in the log as the pattern below"""
+    for tag in verification_tags:
+        if not re.findall(r'(.+Scanning::Job#{}.+)'.format(re.escape(tag)), log):
+            raise Exception('Could not find verification tag in log: {}'
+                            .format(tag))
+
 
+# CMP-9496 CMP-10064 CMP-10007
 
-@pytest.mark.meta(blockers=[1382326, 1406023])
-def test_containers_smartstate_analysis(provider, ssh_client):
+@pytest.mark.meta(blockers=[1382326, 1406023, 1408255, 1371896])
```

Comment:
**OPTIONAL**
BZ 1406023 is in CLOSED ERRATA state and can likely be removed here.